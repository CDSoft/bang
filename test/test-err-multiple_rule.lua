generator(false)

rule "foo" { command = true }
rule "bar" { command = true }
rule "baz" { command = true }
rule "foo" { command = true } -- same rule => no redefinition
rule "foo" { command = false } -- [test/test-err-multiple_rule.lua:6] ERROR: rule foo: multiple definition
