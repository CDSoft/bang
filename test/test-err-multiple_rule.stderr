[test/test-err-multiple_rule.lua:7] ERROR: rule foo: multiple definition
